src/i18n/en.ts

```diff
@@ -55,6 +55,17 @@ export default {
 		"COPY_CODE": "Copy code",
 		"COPIED": "Copied!"
 	},
+	"COMBOBOX": {
+		"PLACEHOLDER": "Filter...",
+		"CLEAR_SELECTIONS": "Clear all selected items",
+		"CLEAR_SELECTED": "Clear selected item",
+		"A11N": {
```

Comment:
Should be

```suggestion
		"A11Y": {
```

Or "ACCESSIBILITY" written out in full

Reply:
LOL. What was i thinking? L11N and A11Y messed up my brain 😆 
Fixed.